tests/unit/test-cmd/test.build.js

```diff
@@ -37,6 +44,94 @@ describe('build', () => {
     );
   });
 
+  it('gives the correct name to an extension that is not localized', () => {
```

Comment:
I don't think you need this test. The same assertion happens in https://github.com/shubheksha/web-ext/blob/8cd7c6659f244094f079e4ddd1ccd46e16bdc5a5/tests/unit/test-cmd/test.build.js#L28-L35 (although, perhaps the title of that test could be updated to say so)